### PR TITLE
build: add release helper scripts

### PR DESCRIPTION
**Pull request by vicb**

### CODE REVIEW

File: scripts/release/post-check-next [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+#!/usr/bin/env bash
+
+# use for PATCH releases
+# Verify that all packages are published with correct version:
+
+mkdir tmp
+cd tmp
+npm init -y
+npm install typescript@2.6 rxjs@5.5.0 zone.js@0.8.10 @angular/{animations,core,common,forms,router,platform-browser,platform-browser-dynamic,platform-webworker,platform-webworker-dynamic,platform-server,compiler,compiler-cli,language-service,service-worker,animations}@next --save
```
Comment (gkalpak) [inline scripts/release/post-check-next]: `animations` appears twice. `upgrade` is missing.

File: scripts/release/post-check-next [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+#!/usr/bin/env bash
+
+# use for PATCH releases
```
Comment (gkalpak) [inline scripts/release/post-check-next]: _PATCH_ --> _BETA and RC_

File: scripts/release/post-check [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+#!/usr/bin/env bash
+
+# use for PATCH releases
+# Verify that all packages are published with correct version:
+
+mkdir tmp
+cd tmp
+npm init -y
+npm install typescript@2.6 rxjs@5.5.0 zone.js@0.8.10 @angular/{animations,core,common,forms,router,platform-browser,platform-browser-dynamic,platform-webworker,platform-webworker-dynamic,platform-server,service-worker,compiler,compiler-cli,upgrade} --save
```
Comment (gkalpak) [inline scripts/release/post-check]: `language-service` is missing. Is that intentional?

On an unrelated note:
Yay! Another list of package names we have to keep up-to-date :tada: 😛

Comment (vicb) [reply scripts/release/post-check]: > Another list of package names we have to keep up-to-date

For now it's maintained in a google doc... incremental improvement :)

File: scripts/release/post-check-next (hunk 1 above)
Comment (vicb) [reply scripts/release/post-check-next]: good catch !

File: scripts/release/post-check (hunk 3 above)
Comment (chuckjaz) [reply scripts/release/post-check]: Yes. The language service should be in this list as well.

File: scripts/release/post-check-next (hunk 2 above)
Comment (vicb) [reply scripts/release/post-check-next]: very good catch !

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**